lib/steps_components/options/OptionsStep.jsx

```diff
@@ -38,7 +38,9 @@ class OptionsStep extends Component {
     return (
       <OptionsStepContainer className="rsc-os">
         <Options className="rsc-os-options">
-          {Object.values(options).map(this.renderOption)}
+          {Object.keys(options).map(key => {
+            return options[key];
```

Comment:
You can remove the return:

```
{Object.keys(options).map(key => options[key]).map(this.renderOption)}
```

Reply:
Updated the PR with the shorthand return. 
Should we add a eslint rule for that?

Reply:
I don't think it's necessary